src/plugins/search/plugin.search.js

```diff
@@ -213,6 +213,7 @@ BookReader.prototype.search = function(term = '', overrides = {}) {
   return $.ajax({
     url: url,
     dataType: 'jsonp',
+    cache: true,
```

Comment:
That's interesting.. is the default behavior to break the cache each time?

Reply:
yes, when `dataType: 'jsonp' https://api.jquery.com/jquery.ajax/